legal-api/src/legal_api/reports/report.py

```diff
@@ -748,6 +751,18 @@ def _format_amalgamation_data(self, filing):
     def _format_certificate_of_amalgamation_data(self, filing):
         self._set_amalgamating_businesses(filing)
 
+    def _format_notice_of_withdrawal_data(self, filing):
```

Comment:
This may not be enough for T business. Did you test this with T business?

Reply:
Not at this moment, as the legal-api is not yet supporting filing NoW for temporary businesses. And as mentioned above, the T business part will be covered in https://github.com/bcgov/entity/issues/24988